### PR TITLE
Find items by part_of_item uuid

### PR DESCRIPTION
**Pull request by clorenz**

### CODE REVIEW

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java [hunk 1]
```diff
@@ -214,6 +220,22 @@ private UUID[] extractHolderUuids(Item item) {
...
   }
 
+  @Override
+  public String getColumnName(String modelProperty) {
+    if (modelProperty == null) {
+      return null;
+    }
+    if (super.getColumnName(modelProperty) != null) {
+      return super.getColumnName(modelProperty);
+    }
```
Comment (daforster) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java:230]:
```suggestion
```

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java [hunk 2]
```diff
@@ -214,6 +220,22 @@ private UUID[] extractHolderUuids(Item item) {
...
+    switch (modelProperty) {
+      case "part_of_item.uuid":
+        return tableAlias + ".part_of_item";
+      default:
+        return null;
```
Comment (daforster) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java:235]:
```suggestion
        return super.getColumnName(modelProperty);
```

**Review by daforster — CHANGES_REQUESTED** [review]

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java (hunk 1 above)
Comment (clorenz) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java]: done

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java (hunk 2 above)
Comment (clorenz) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/ItemRepositoryImpl.java]: done

**Review by daforster — APPROVED** [review]

**Pull request merged**